src/docfx/build/legacy/metadata/LegacyMetadata.cs

```diff
@@ -120,46 +118,9 @@ public static JObject GenerateLegacyRedirectionRawMetadata(Docset docset, PageMo
                     rawMetadata["original_content_git_url"] = pageModel.OriginalContentGitUrl;
             }
 
-            return RemoveUpdatedAtDateTime(LegacySchema.Transform(Jint.Run(rawMetadata), pageModel)).RemoveNulls();
-        }
-
-        public static string GenerateLegacyPageMetadata(JObject rawMetadata)
-        {
-            StringBuilder pageMetadataOutput = new StringBuilder(string.Empty);
-
-            foreach (var item in rawMetadata)
-            {
-                if (!s_pageMetadataBlackList.Any(blackList => item.Key.StartsWith(blackList)))
-                {
-                    string key = HttpUtility.HtmlEncode(item.Key);
-                    string content;
-                    if (item.Value is JArray)
-                    {
-                        content = string.Join(",", item.Value);
-                    }
-                    else if (item.Value.Type == JTokenType.Boolean)
-                    {
-                        content = (bool)item.Value ? "true" : "false";
-                    }
-                    else
-                    {
-                        content = item.Value.ToString();
-                    }
-
-                    if (key.Equals("robots", StringComparison.OrdinalIgnoreCase))
-                    {
-                        // Add to the first line
-                        key = "ROBOTS";
-                        pageMetadataOutput.Insert(0, $"<meta name=\"{key}\" content=\"{HttpUtility.HtmlEncode(content)}\" />{Environment.NewLine}");
-                    }
-                    else
-                    {
-                        pageMetadataOutput.AppendLine($"<meta name=\"{key}\" content=\"{HttpUtility.HtmlEncode(content)}\" />");
-                    }
```

Comment:
Missed this part after move?

Reply:
this is intentional, the order and casing does not matter in html <meta> tags. If it is for comparison, it should be done in patch tool.

Reply:
Another request is to make the "robots" upper case.

Reply:
sitemap requires it to be either `robots` or `ROBOTS`. Treat all metadata keys case insensitive. And make 'robots' as a known metadata for build.